bcm4908: ImageBuilder: asus,gt-ac5300/cferam.000': No such file or directory

### Describe the bug

https://github.com/openwrt/docker/actions/runs/14251313226/job/39944529737#step:7:1006

```shell
vol_flags=autoresize
cp: cannot stat '/builder/build_dir/target-aarch64_cortex-a53_musl/linux-bcm4908_generic/bcm63xx-cfe/asus,gt-ac5300/cferam.000': No such file or directory
make[3]: *** [Makefile:140: /builder/build_dir/target-aarch64_cortex-a53_musl/linux-bcm4908_generic/tmp/openwrt-bcm4908-generic-asus_gt-ac5300-squashfs.bin] Error 1
make[2]: *** [Makefile:263: build_image] Error 2
make[1]: *** [Makefile:159: _call_image] Error 2
make: *** [Makefile:332: image] Error 2
```

### OpenWrt version

Snapshot

### OpenWrt release

Snapshot

### OpenWrt target/subtarget

bcm4908/generic

### Device

ASUS GT-AC5300

### Image kind

Official downloaded image

### Steps to reproduce

_No response_

### Actual behaviour

_No response_

### Expected behaviour

_No response_

### Additional info

_No response_

### Diffconfig

```text

```

### Terms

- [x] I am reporting an issue for OpenWrt, not an unsupported fork.

--- a/package/kernel/bcm63xx-cfe/Makefile
+++ b/package/kernel/bcm63xx-cfe/Makefile
@@ -35,9 +35,9 @@ define Package/bcm63xx-cfe/install
 endef
 
 define Build/InstallDev
-	rm -rf $(KERNEL_BUILD_DIR)/$(PKG_NAME)
-	mkdir -p $(KERNEL_BUILD_DIR)/$(PKG_NAME)
-	$(CP) -r $(PKG_BUILD_DIR)/* $(KERNEL_BUILD_DIR)/$(PKG_NAME)
+	rm -rf $(STAGING_DIR_IMAGE)/$(PKG_NAME)
+	mkdir -p $(STAGING_DIR_IMAGE)/$(PKG_NAME)
+	$(CP) -r $(PKG_BUILD_DIR)/* $(STAGING_DIR_IMAGE)/$(PKG_NAME)
 endef
 
 $(eval $(call BuildPackage,bcm63xx-cfe))

--- a/target/linux/bcm4908/image/Makefile
+++ b/target/linux/bcm4908/image/Makefile
@@ -33,7 +33,7 @@ define Build/bcm4908img
 	cp -r $(DEVICE_NAME)/* $@-bootfs/
 	touch $@-bootfs/1-openwrt
 	cp $(DTS_DIR)/$(firstword $(DEVICE_DTS)).dtb $@-bootfs/94908.dtb
-	cp $(KDIR)/bcm63xx-cfe/$(subst _,$(comma),$(DEVICE_NAME))/cferam.000 $@-bootfs/
+	cp $(STAGING_DIR_IMAGE)/bcm63xx-cfe/$(subst _,$(comma),$(DEVICE_NAME))/cferam.000 $@-bootfs/
 	cp $(IMAGE_KERNEL) $@-bootfs/vmlinux.lz
 
 	$(STAGING_DIR_HOST)/bin/mkfs.jffs2 --pad=0x800000 --little-endian --squash-uids \

--- a/target/linux/bmips/image/Makefile
+++ b/target/linux/bmips/image/Makefile
@@ -136,7 +136,7 @@ define Build/cfe-jffs2-cferam
 	# will have version 0 and let cferam be the second (version 1).
 	touch $@-cferam/1-openwrt
 	# Add cferam as the last file in the JFFS2 partition
-	cp $(KDIR)/bcm63xx-cfe/$(CFE_RAM_FILE) $@-cferam/$(CFE_RAM_JFFS2_NAME)
+	cp $(STAGING_DIR_IMAGE)/bcm63xx-cfe/$(CFE_RAM_FILE) $@-cferam/$(CFE_RAM_JFFS2_NAME)
 
 	# The JFFS2 partition creation should result in the following
 	# layout:

